### PR TITLE
exit on state 127

### LINKED ISSUE #76: SQLCMD should exit on certain errors
Consider:

RAISERROR('This is the end', 1, 127)

When you run this in the old SQLCMD, the scripts exits, because state is 127. go-sqlcmd does not. (This an old holdover from ISQL, and should still work I think. No, this does not work in SSMS.)

On a similar note:

RAISERROR('Get me out of here!', 20, 1) WITH LOG

This closes the connection to SQL Server, and old SQLCMD exits at this point. go-sqlcmd does not, but nor does it reconnect, so you only get more error messages.

### PR DESCRIPTION
**Pull request by shueybubbles**
Fixes #76 
Had to pick up new `sqlexp` and `go-mssqldb`

### CODE REVIEW

**Comment by shueybubbles** [comment]
also fixes #87 (I hope)

**Comment by shueybubbles** [comment]
Fixes #39

File: pkg/sqlcmd/sqlcmd.go [hunk 1]
```diff
@@ -44,6 +44,8 @@ type Console interface {
 	ReadPassword(prompt string) ([]byte, error)
 	// SetPrompt sets the prompt text shown to input the next line
 	SetPrompt(s string)
+	// Close clears any buffers and closes open file handles
+	Close()
```
Comment (stuartpa) [inline pkg/sqlcmd/sqlcmd.go:48]: I'm not seeing an implementation for this method, is this by design?

**Review by stuartpa — APPROVED** [review]

File: pkg/sqlcmd/sqlcmd.go (hunk 1 above)
Comment (shueybubbles) [reply pkg/sqlcmd/sqlcmd.go:48]: it's already implemented. https://github.com/microsoft/go-sqlcmd/blob/e3972914801bc3492b49b6fe642a7e931e088be3/pkg/console/console.go#L39

**Pull request merged**